pkgs/applications/networking/cluster/helmsman/default.nix

```diff
@@ -13,6 +13,8 @@ buildGoModule rec {
 
   vendorSha256 = "05vnysr5r3hbayss1pyifgp989kjw81h95iack8ady62k6ys5njl";
 
+  doCheck = false;
```

Comment:
How do you plan to enable them?

Reply:
By checking them and making PRs but not all at once. I wanted this to be a no-rebuild PR so it's an easy merge.

I wanted to flip the default now while it isn't to much trouble and so it applies to `buildGoPackage` -> `buildGoModule` changes and new packages.

Reply:
Have you checked what the success rate for go packages in general is. I am asking because this will also affect out-of-tree packages.

Reply:
Currently it's barely used and doesn't get mentioned during PR reviews.

Making this out out (same as `buildRustPackage`) will be better long term.

> I am asking because this will also affect out-of-tree packages.

Yeah, I think now is a good time to do this with `vendorSha` changes also affecting those packages.